livy/README.md

```diff
@@ -0,0 +1,26 @@
+# Apache Livy Initialization Action
```

Comment:
Maybe add a TODO to the HUE init action to integrate with this (or just file an issue)

Reply:
Ack

Reply:
Ack